patches/linux/linux64.patch

```diff
@@ -1,19 +1,12 @@
 diff --git arch/riscv/mm/init.c arch/riscv/mm/init.c
-index 42bf939693d3..928d5d80f103 100644
+index d466ec670e1f..2454223f3f76 100644
 --- arch/riscv/mm/init.c
 +++ arch/riscv/mm/init.c
-@@ -18,6 +18,8 @@
- #include <asm/pgtable.h>
- #include <asm/io.h>
- 
-+#include <linux/dma-contiguous.h>
-+
- unsigned long empty_zero_page[PAGE_SIZE / sizeof(unsigned long)]
- 							__page_aligned_bss;
- EXPORT_SYMBOL(empty_zero_page);
-@@ -444,4 +446,5 @@ void __init paging_init(void)
- 	setup_vm_final();
- 	setup_zero_page();
+@@ -1210,6 +1210,7 @@ void __init misc_mem_init(void)
  	zone_sizes_init();
+ 	reserve_crashkernel();
+ 	memblock_dump_all();
 +	dma_contiguous_reserve(memblock_end_of_DRAM());
```

Comment:
The CMA is available in latest linux kernel, so we do not need this patch anymore. (add_patch was deleted). Please remove this file!

Reply:
I have removed this file.